src/server/utilities/cspHeader/index.js

```diff
@@ -266,6 +282,22 @@ const directives = {
       'https://passport-control.int.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - int
       'https://passport-control.test.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - test
       'https://passport-control.tools.bbc.co.uk/bookmarkletScript.js', // Passport bookmarklet - live
+      // Ads
+      'https://ad.crwdcntrl.net',
+      'https://adservice.google.co.uk',
+      'https://adservice.google.com',
+      'https://bbc.gscontxt.net',
+      'https://bcp.crwdcntrl.net',
+      'https://cdn.ampproject.org',
+      'https://collector.effectivemeasure.net',
+      'https://me-ssl.effectivemeasure.net',
+      'https://privacy.crwdcntrl.net',
+      'https://sb.scorecardresearch.com',
+      'https://securepubads.g.doubleclick.net',
+      'https://t.effectivemeasure.net',
+      'https://tags.crwdcntrl.net',
+      'https://tpc.googlesyndication.com',
+      'https://gn-web-assets.api.bbc.com',
```

Comment:
When running locally with `npm run build && npm run start` & visiting http://localhost:7080/afrique, I saw CSP header errors, so I've suggested these additions, which fix the script-src errors:
```suggestion
      'https://gn-web-assets.api.bbc.com', // ads
      'https://pixel.adsafeprotected.com', // ads
      'https://static.adsafeprotected.com', // ads
      'https://www.googletagservices.com', // ads
```

Reply:
Additionally, to make the CSP headers not show, the following need to be added to the 
imgSrc, canonicalNotLive list:
```
      'https://pixel.adsafeprotected.com', // ads
      'https://dt.adsafeprotected.com', // ads
```